doc/examples/motion_planning_python_api/scripts/motion_planning_python_api_tutorial.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+"""
+A script to outline the fundamentals of the moveit_py motion planning API.
+"""
+import numpy as np
+import rclpy
+from rclpy.node import Node
+import sys
+import threading
+import time
+
+from geometry_msgs.msg import Pose, PoseStamped
+from moveit_msgs.msg import Constraints
+
+from moveit.planning import (
+    MoveItPy,
+    PlanRequestParameters,
+    MultiPipelinePlanRequestParameters,
+)
+from moveit.core.robot_state import RobotState
+from moveit.core.kinematic_constraints import construct_joint_constraint
+
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+import yaml
+
+
+def main():
+
+    ###################################################################
+    # MoveItPy Setup
+    ###################################################################
+    rclpy.init()
+    logger = rclpy.logging.get_logger("moveit_py.pose_goal")
+
+    # instantiate MoveItPy instance and get planning component
+    moveit = MoveItPy(node_name="moveit_py")
+    panda_arm = moveit.get_planning_component("panda_arm")
+    logger.info("MoveItPy instance created")
+
+    ###########################################################################
+    # Plan 1 - set states with predefined string
+    ###########################################################################
+
+    # set plan start state using predefined state
+    panda_arm.set_start_state(configuration_name="ready")
+
+    # set pose goal using predefined state
+    panda_arm.set_goal_state(configuration_name="extended")
+
+    # plan to goal
+    logger.info("Planning trajectory")
+    plan_result = panda_arm.plan()
+
+    # execute the plan
+    if plan_result:
+        logger.info("Executing plan")
+        robot_trajectory = plan_result.trajectory
+        moveit.execute(robot_trajectory, blocking=True, controllers=[])
```

Comment:
I'm getting spammed a lot with controller manager related things. 

```
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016806322] [moveit_py.pose_goal]: Executing plan
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016868326] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016914273] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016934701] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016939193] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.016953487] [moveit_ros.trajectory_execution_manager]: Validating trajectory with allowed_start_tolerance 0.01
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.017974801] [moveit_ros.trajectory_execution_manager]: Starting trajectory execution ...
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.018007381] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
[motion_planning_python_api_tutorial.py-4] [INFO] [1677107245.018020536] [moveit.plugins.moveit_simple_controller_manager]: Returned 2 controllers in list
```

This happens whenever you call `execute()` through the `getControllersList()` function.

Not sure what the best approach is here. Could it be to change the log level of the Python tutorial from INFO to something higher like WARN/ERROR, and then have all the logger messages in the tutorial itself be regular print statements? Or do we just live with it?

If the change is too complicated, I'd say forget it for now

Reply:
> Could it be to change the log level of the Python tutorial from INFO to something higher like WARN/ERROR, and then have all the logger messages in the tutorial itself be regular print statements?

I would be in favour of this approach for the tutorial.